README.md

```diff
@@ -8,9 +8,20 @@ Derelict|Prisoner
 -|-
 ![](media/carousel.png) | ![](media/prisoner.png)
 
+# Build guide
+```
+$ git clone https://github.com/Wunkolo/vkblam && cd vkblam
+$ git clone https://github.com/vector-of-bool/cmrc external/cmrc
+$ git clone https://github.com/mandreyel/mio external/mio
+$ git clone https://github.com/g-truc/glm external/glm
```

Comment:
This can be done with
```
git clone --recursive https://github.com/Wunkolo/vkblam
cd vkblam
```

Reply:
![image](https://user-images.githubusercontent.com/12180913/187024346-ffbd53e3-411c-4b7c-a6f3-816433f70456.png)
I couldn't clone the sub-modules due to presumably a configuration issue. I honestly have never needed to work with them before today so I'm unsure how to correct that problem.

Reply:
Fixed in [bbc8649](https://github.com/Wunkolo/vkblam/pull/1/commits/bbc8649cf4c93550cbd1a16611d7e3a3c108c38b) and [f8b1f0e](https://github.com/Wunkolo/vkblam/pull/1/commits/f8b1f0e4b15c1cdc7e90eca557d20fa522826ec1). Closing.